eng/pipelines/templates/variables/image.yml

```diff
@@ -2,18 +2,18 @@
 
 variables:
   - name: LINUXPOOL
-    value: azsdk-pool-mms-ubuntu-2004-general
+    value: azsdk-pool
   - name: WINDOWSPOOL
-    value: azsdk-pool-mms-win-2022-general
+    value: azsdk-pool
   - name: MACPOOL
     value: Azure Pipelines
 
   - name: LINUXVMIMAGE
-    value: azsdk-pool-mms-ubuntu-2004-1espt
-  - name: LINUXNEXTVMIMAGE
     value: ubuntu-22.04
```

Comment:
Any reason to leave this one behind?

Reply:
Because they were actively testing on `ubuntu 22.04`, so I updated to include `24.04` as well.

Reply:
@heaths I assume you guys need to explicitly test on 22.04?

Reply:
They don't need it critically but would prefer to maintain it if it costs nothing. They don't use a huge amount of agents so I don't see any harm in keeping it around here.